README.md

```diff
@@ -0,0 +1,22 @@
+# ruby-hangman-pair
```

Comment:
👏 Woo hoo! Readme! These are often forgotten, and are way better to write while working on the project, rather than looking back and saying "eek, I've gotta write 13 READMEs 😂"